ports/atmel-samd/boards/trinket_m0_haxpress/mpconfigboard.h

```diff
@@ -11,11 +11,6 @@
 #define SPI_FLASH_SCK_PIN           &pin_PA17
 #define SPI_FLASH_CS_PIN            &pin_PA11
 
-// These are pins not to reset.
-#define MICROPY_PORT_A        (PORT_PA00 | PORT_PA01 | PORT_PA18)
```

Comment:
PA00 and PA01 are the APA102 pins, but they are commented out on lines 6 and 7. Is that a mistake? They are not commented out in the regular `trinket_m0/mpconfigboard.h`

Reply:
I don't know. I assumed they may share flash pins.

Reply:
I looked at the schematic, and PA00 and PA01 are still just connected to the NeoPixel. So I think those lines should be uncommented.
![image](https://user-images.githubusercontent.com/2847802/125874637-aa20af16-3c54-4764-8f9b-5c06232ee20a.png)